src/ServicePulse.Host/vue/src/components/CustomCheckView.vue

```diff
@@ -0,0 +1,41 @@
+<script setup lang="ts">
```

Comment:
should be called CustomChecks or CustomChecksList, not CustomChecksView

Reply:
It's the view of a single custom check. I originally called it `CustomCheck` but that conflicted with the type in the `Resources` folder.